tests/main/interfaces-timeserver-control/task.yaml

```diff
@@ -4,6 +4,9 @@ details: |
     This test makes sure that a snap using the timeserver-control interface
     can access timeserver information and update it.
 
+# Debian sid is skipped because "timedatectl set-ntp" fails
```

Comment:
Can you please give more information about how it fails. Sit should be very similar to Ubuntu so this may be an upcoming error that will hit us everywhere.